### PR TITLE
[Dungeon] Blackfathom Deeps

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

**Comment by Geekena** [comment]
damage and spells are ok 
no problems

**Comment by Helias** [comment]
@Geekena thank you for testing these several PRs!

File: data/sql/updates/pending_db_world/rev_1588338975542081700.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.02 WHERE `entry` = 12902;
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 86, `maxdmg` = 110, `DamageModifier` = 1.02 WHERE `entry` = 12876;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 94, `maxdmg` = 108, `DamageModifier` = 1.01 WHERE `entry` = 4887;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 112, `maxdmg` = 144, `minrangedmg` = 28, `maxrangedmg` = 42, `DamageModifier` = 1.01 WHERE `entry` = 4831;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 6243;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 4830;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 4832;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 4829;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588338975542081700.sql:44]: type_flags

**Comment by Geekena** [comment]
not a problem
only time cause kids :)

File: data/sql/updates/pending_db_world/rev_1588338975542081700.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1588338975542081700.sql]: Sorted mate.

**Pull request merged**